src/coretelephony.cs

```diff
@@ -69,9 +69,11 @@ interface CTRadioAccessTechnology {
 
 	[BaseType (typeof (NSObject))]
 	interface CTTelephonyNetworkInfo {
+		[Deprecated (PlatformName.iOS, 12,0, message: "Use 'ServiceSubscriberCellularProviders' instead.")]
 		[Export ("subscriberCellularProvider", ArgumentSemantic.Retain)]
 		CTCarrier SubscriberCellularProvider { get; }
 
+		[Deprecated (PlatformName.iOS, 12,0, message: "Use 'CellularProvidersDidUpdateNotifier' instead.")]
```

Comment:
This is now: `ServiceSubscriberCellularProvidersDidUpdateNotifier`.

👀  https://github.com/xamarin/xamarin-macios/pull/4803#discussion_r217233430 👀